src/vmm/src/vstate/vcpu/aarch64.rs

```diff
@@ -103,6 +88,7 @@ impl KvmVcpu {
         &mut self,
         guest_mem: &GuestMemoryMmap,
         kernel_load_addr: GuestAddress,
+        vcpu_config: &VcpuConfig,
```

Comment:
worth mentioning the new argument in the function's doc.

Reply:
done